Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -135,13 +150,96 @@ - (IBAction)showCertInfo:(id)sender {
                                                showGroup:YES];
 }
 
+// When running in standalone mode, the user is prompted to approve the binary.
+- (void)approveBinaryForStandaloneMode {
+  LAContext *context;
+  NSError *err;
+
+  dispatch_semaphore_t sema = dispatch_semaphore_create(0);
+  context = [[LAContext alloc] init];
+
+  if (![context canEvaluatePolicy:LAPolicyDeviceOwnerAuthenticationWithBiometrics error:&err]) {
```

Comment:
I would do this check even before changing the UI in -loadWindow. Or better yet, when standalone mode is enabled, but bio auth is not available, we display an error in the block popup letting the user know.

Reply:
> Or better yet, when standalone mode is enabled, but bio auth is not available, we display an error in the block popup letting the user know

This seems like a good idea. Should we also disable the `Approve` button so users don't get frustrated?

Reply:
Yeah, if the desired auth is not available we shouldn't offer standalone buttons.

Reply:
I made the button disappear. But haven't figured out how to update the UI to display to the user that things are not operating as intended.